.eslintrc.yml

```diff
@@ -0,0 +1,10 @@
+env:
+  browser: true
```

Comment:
Do we need this?

Reply:
When I was selecting the options, browser was always selected, even tho I switched to Node. I will delete this.